website/project/views/comment.py

```diff
@@ -139,6 +139,32 @@ def send_comment_added_notification(comment, auth):
             )
 
 
+@mention_added.connect
+def send_mention_added_notification(comment, new_mentions, auth):
+    node = comment.node
+    target = comment.target
+
+    context = dict(
+        gravatar_url=auth.user.profile_image_url(),
+        content=markdown.markdown(comment.content, ['del_ins', 'markdown.extensions.tables', 'markdown.extensions.fenced_code']),
```

Comment:
Minor: there's some duplication with this line: https://github.com/CenterForOpenScience/osf.io/blob/fe718f7ef94486949c7fe996ac07ea07c46a3ea0/website/project/views/comment.py#L114 . It would be useful to hide implementation details, e.g. which markdown plugins we're using:

```python
content=render_email_markdown(comment.content)
```